cli/cmd/generate.go

```diff
@@ -30,6 +30,30 @@ func (a *AwsGenerateCommandExtraState) writeCache() {
 	}
 }
 
+type GcpGenerateCommandExtraState struct {
```

Comment:
Would it be better to have the extra state definition and associated function in the same file as the generate args definition, eg in the lwgenerate/gcp.go file rather than the 'generic' generate.go file

Reply:
I would agree with this suggestion 👆🏽